nativeruntime/cpp/CMakeLists.txt

```diff
@@ -132,5 +121,8 @@ if (CMAKE_HOST_SYSTEM_NAME MATCHES "Linux")
   target_link_libraries(nativeruntime
     -static-libgcc
     -static-libstdc++
+    -ldl
+    -lpthread
+    -Wl,--no-undefined # print an error if there are any undefined symbols
```

Comment:
Moving `--no-undefined` to a new line to follow above code style?

Reply:
`-Wl,--no-undefined` is actually meant to be on the same line. `-Wl` means "use the following linker option". It has the format `-Wl,<OPTION>`

Reply:
Thanks for your response.